webapp-django/crashstats/supersearch/form_fields.py

```diff
@@ -116,6 +118,12 @@ def value_to_string(self, value):
         except AttributeError:  # when value is None
             return value
 
+    def strptime(self, value, format):
```

Comment:
Ah, this is fixing a bug that I found as a side effect of this change. It changes the behavior of the ``DatetimeField`` class from django to use isodate.parse_datetime as a first option whenever trying to parse a datetime. By default, django doesn't understand isoformat dates (the ones with a ``T`` in the middle) and that's bothering me. I figured the easiest way to deal with that in the current system is to overwrite this method to use ``isodate``. 

An other solution was to extend the list of accepted date formats passed to this field, but in the mess of heritage around here, I failed to do that.